pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -1559,12 +1560,54 @@ private boolean isTmpAndCanDelete(URI uri, Set<String> deepURIs, PinotFS pinotFS
 
   /**
    * Force commit the current segments in consuming state and restart consumption
+   * Commit all partitions unless either partitionsToCommit or segmentsToCommit are provided.
+   *
+   * @param tableNameWithType  table name with type
+   * @param partitionsToCommit  comma separated list of partitions to commit
```

Comment:
To be consistent, should it be "partitionGroupIdsToCommit" instead of "partitionsToCommit"?